### ISSUE TITLE
venv selection fails

### ISSUE THREAD

**Issue by priamai** [jupyterlab/jupyterlab-desktop#603]
## Description
Upon selecting an environment and choosing a venv python one.
I am getting this error.

![image](https://user-images.githubusercontent.com/57333254/222949183-78804067-5f54-4f99-9dec-f237691fb90c.png)

## Reproduce

Create a virtual environment with venv and then select it.

**Comment by mbektas** [comment]
@priamai do you have `jupyterlab` Python package installed in your custom venv? you can install using `pip install jupyterlab`.

**Comment by JasonWeill** [comment]
@priamai Closing this issue due to inactivity. Please reopen with an answer to @mbektas 's question or open a new issue if you still notice this after installing `jupyterlab` in your venv. Thanks!

**Comment by priamai** [comment]
Testing now, sorry I missed the message.

**Comment by priamai** [comment]
Installed it

```
(venv) robomotic@priam-thinkpad:~/DevOps/github/firstorg2023/lib$ pip show jupyterlab
Name: jupyterlab
Version: 3.6.3
Summary: JupyterLab computational environment
Home-page: https://jupyter.org
Author: Jupyter Development Team
Author-email: jupyter@googlegroups.com
License: 
Location: /home/robomotic/DevOps/github/firstorg2023/venv/lib/python3.8/site-packages
Requires: ipython, jinja2, jupyter-core, jupyter-server, jupyter-server-ydoc, jupyter-ydoc, jupyterlab-server, nbclassic, notebook, packaging, tomli, tornado
Required-by: 
```
But when I select the related venv still get the error:
![image](https://user-images.githubusercontent.com/57333254/229951860-925d490a-dd4f-4a94-80ce-f7378385fe03.png)

**Comment by priamai** [comment]
@JasonWeill can you reopen please see my message above.

**Comment by mbektas** [comment]
@priamai the environment path which you ran `pip show jupyterlab` and the one in your screenshot are different.
`/home/robomotic/DevOps/github/firstorg2023/venv/` vs
`/home/robomotic/DevOps/firstorg2023/venv/`

**Comment by priamai** [comment]
silly me, yes wrong path this time, it works fine with the right path. one suggestion would be to make the error more significant, maybe say folder path does not exist instead of Invalid Environment? and what about if the path is correct also remind the user that it needs to install the right packages? I know its' in the documentation but I didn't read that until I had the problem :-)

**Comment by mbektas** [comment]
closing since the issue resolved
